testpilot/frontend/urls.py

```diff
@@ -1,8 +1,10 @@
-from django.conf.urls import url, patterns
+from django.conf.urls import patterns, url
 
 from . import views
 
+
 urlpatterns = patterns(
     '',
-    url(r'(?P<url>.*)', views.index, name="frontend_index")
+    url(r'^contribute\.json$', views.contribute_json, name='contribute_json'),
```

Comment:
Ok, thanks. My eyes jumped right over "base" when I was looking for reasonable place to put this.